pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -220,6 +159,112 @@ def run_func(self, *args, **kwargs):
 
         return decorator
 
+    def _get_pattern_lst(self, source_path, timestamps=False):
+        '''
+        Get an order list follow the name pattern
+        '''
+        if timestamps:
+            pattern, ext = os.path.basename(
+                os.path.normpath(source_path)).split(".")
+            suffix = "_timestamps.npy"
+        else:
+            pattern, suffix = os.path.basename(
+                os.path.normpath(source_path)).split(".")
+            suffix = "." + suffix
+        path = []
+        for file in os.listdir(os.path.dirname(source_path)):
+            if fnmatch.fnmatch(file, pattern + "*" + suffix):
+                path.append(file)
```

Comment:
I would recommend to use the https://docs.python.org/3/library/glob.html#module-glob package for this kind of work

Reply:
Ok, I will update it.